### ISSUE TITLE
move from youtube-dl to yt-dlp

### ISSUE THREAD

**Issue by Lunarequest** [JaGoLi/ytdl-gui#61]
youtube-dl has not had commits in a very long time and the project maintainer has not communicated with anyone for a very long time. The community has created yt-dlp a spiritual successor to youtube-dl adding many features to the original code base include sponsor block api integration 

https://github.com/yt-dlp/yt-dlp

**Comment by stevenpusser** [comment]
Does it work if you symlink youtube-dl to the youtube-dlp executable?

**Comment by ghost** [comment]
> Does it work if you symlink youtube-dl to the youtube-dlp executable?

It should work: https://github.com/jely2002/youtube-dl-gui/issues/151#issuecomment-903100478

**Comment by SwampRabbit** [comment]
I can confirm that at least on MX-21 (Debian 11) that replacing the runtime dependency to youtube-dlp works.  We have a youtube-dl transitional package though included with yt-dlp.

**Comment by JaGoLi** [comment]
Good to know that it works! My current issue with yt-dlp is that it is not packaged properly in many distros. It is not in the standard archlinux repositories and is not in debian either. The solution to this will be to remove youtube-dl dependencies from any packaging so that a user can install whatever program they want and symlink as @stevenpusser suggests.

**Comment by mbnoimi** [comment]
[yt-dlp](https://github.com/yt-dlp/yt-dlp) no longer alien anymore. It has 30.5k stars (youtube-dl has 113k and decreasing) in Github and used by many GUI clients.
It's your desicion to use it or not but believe me ignoring it is a damn fault decision because `yt-dlp` shipped by many nice features you can't find it in recent `youtube-dl` version

**Comment by Lunarequest** [comment]
Yup yt-dlp is also now in the arch repos. I've thought about moving the flatpak to it but it's got a ton of deps

**Comment by sebma** [comment]
@JaGoLi `yt-dlp` is now in the [debian repositories since the stable-backports](https://packages.debian.org/search?keywords=yt-dlp) and now in the [ArchLinux community repositories](https://archlinux.org/packages/community/any/yt-dlp/). 

Can you please reconsider and reopen this issue ?

**Comment by JaGoLi** [comment]
I haven't had time to work on this project, but since it's now in the debian and archlinux repos it's worth porting it over. So, I'm reopening the issue.

**Comment by sebma** [comment]
@JaGoLi Thanks a lot.
